src/Compilers/CSharp/Test/Emit/Emit/DynamicAnalysis/DynamicInstrumentationTests.cs

```diff
@@ -254,6 +254,7 @@ static void TestMain()
 Flushing
 1
 True
+True
```

Comment:
@JohnHamby Would you like me to restore the original behavior of the code coverage instrumentation?